pkgs/top-level/python-packages.nix

```diff
@@ -6703,6 +6703,8 @@ in {
 
   plaid-python = callPackage ../development/python-modules/plaid-python { };
 
+  plantuml = callPackage ../development/python-modules/plantuml { };
+
   plaster = callPackage ../development/python-modules/plaster { };
```

Comment:
Looks like you'll need update `master` and rebase, it looks like there is a conflict in this part.